app/views/admin/people/new.html.erb

```diff
@@ -1,12 +1,16 @@
-<% page_title "New person" %>
+<% content_for :page_title, "New person" %>
+<% content_for :title, "New person" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @person)) %>
 
-<div class="row">
-  <section class="col-md-8">
-    <h1>New person</h1>
-    <p class="warning">Do not create ministerial people without consulting GDS.</p>
-    <%= render partial: "form", locals: {person: @person, show_consult_gds_warning: true} %>
-  </section>
-  <div class="col-md-4">
-    <%= legacy_simple_formatting_sidebar %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Do not create ministerial people without consulting GDS.",
```

Comment:
Expand acronym for clarity

Reply:
I'm not sure a pattern has been established within Whitehall's UI for expanding acronyms yet. For example, other areas in the 'Machinery of Government' have similar text which mention 'GDS' without expansion. Likewise the Dashboard page includes a link to the "GDS support form (Zendesk)".

I'm fairly confident that anybody using Whitehall will already be aware of what the acronym 'GDS' means – they will already have interacted with GDS through publisher training courses and the like.

So for now, I think it's safe to leave this as is. At least until the team has a Content Designer who's able to iterate the UI copy beyond like-for-like.